cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -317,6 +326,7 @@ def _configure_dovecot(config: Config, debug: bool = False) -> bool:
         mode="644",
         config=config,
         debug=debug,
+        listen_ipv4_only=listen_ipv4_only,
```

Comment:
```suggestion
        listen_ipv4_only="listen = *" if config.disable_ipv6 else "",
```

Then you can do nice little things like this, and even leave out the 3 lines added above...

Reply:
done